README.md

```diff
@@ -1,4 +1,5 @@
 # Goblint
+
```

Comment:
There are still some unrelated changes in the PR diff.

Reply:
Should be fixed now